src/main/java/homework_6/map_problems_generator/MutableClass.java

```diff
@@ -0,0 +1,32 @@
+package homework_6.map_problems_generator;
+
+import lombok.Data;
+
+import java.util.Calendar;
+import java.util.Random;
+
+@Data
+public class MutableClass {
+    private Calendar dateOfBirth;
+
+    MutableClass(Calendar dateOfBirth) {
+        this.dateOfBirth = dateOfBirth;
+    }
+
+    @Override
+    public int hashCode() {
+        Random random = new Random();
+        return random.nextInt();
```

Comment:
We looked at this problem on the lecture, right?

Reply:
Fixed (object's hashcode tied to the field's hashcode (date of birth)).